hawkular-alerts-api/src/main/java/org/hawkular/alerts/api/model/action/Action.java

```diff
@@ -27,7 +29,7 @@
  * A Trigger definition can be linked with a list of actions.
  *
  * Alert engine only needs to know an action id and message/payload.
- * Action payload can optionally have an alert as payload.
+ * Action payload must have an alert as payload.
```

Comment:
Yes, as this work was done in the master branch with no notion of Event, I didn't want to mix concepts.
We will refactor once we can apply the Events branch.